packages/admin/admin/src/actionLog/diffHeader/DiffHeader.sc.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from "@emotion/react";
```

Comment:
Generally we import this from MUI (not sure if this actually makes a difference but we'd be more consistent)

```suggestion
import { css } from "@mui/material/styles";
```

Reply:
updated this.